tests/unit/mock-api-data/text_androids.json

```diff
@@ -0,0 +1,1149 @@
+[
+  {
+    "text": "do androids dream of electric sheep",
+    "score": null,
+    "pos_types": null,
+    "fingerprint": {
+      "positions": [
```

Comment:
That's fine, but the mock JSON file should be the response as it is returned from the API, not including any local mutations. Asserting that the structure is updated from that raw response by the code under test would be part of the test assertions.